source/tutorials/nixos/nixos-configuration-on-vm.md

```diff
@@ -180,13 +180,19 @@ ls -R ./result
     run-nixos-vm
 ```
 
-Run the virtual machine:
+To run the virtual machine in a new window (requires GUI environment):
 
 ```shell-session
 ./result/bin/run-nixos-vm
 ```
 
-This command opens a window that shows the boot process of the virtual machine and ends at the `gdm` login screen where you can log in as `alice` with the password `testpw`.
+To run it in the current terminal instead, pass `-nographic`:
```

Comment:
Generally, instead of presenting multiple options - which is distracting to first-time users because they'll be busy with following *one* set of instructions, and also doesn't scale well, since where do you draw the line? – we should link to the manual right afterward, such as:

```
Refer to the Nixpkgs manual for a [list of all commmand-line options to `run-nixos-vm`](<URL>).
```

(If that exists. Otherwise there's likely a `--help` flag?)